ci-operator/step-registry/cucushift/installer/rehearse/azure/ipi/multidisk/provision/cucushift-installer-rehearse-azure-ipi-multidisk-provision-chain.yaml

```diff
@@ -0,0 +1,13 @@
+chain:
+  as: cucushift-installer-rehearse-azure-ipi-multidisk-provision
```

Comment:
I guess this install can be covered by `cucushift-installer-rehearse-azure-ipi-multidisk-swap-provision`, just not set `AZURE_CONTROL_PLANE_MULTIDISK_TYPE` to `swap` ?

Reply:
hmm, yes if we merge step `ipi-conf-azure-multidisk-swap` to `ipi-conf-azure-multidisk`

Reply:
Done